### ISSUE TITLE
[Bug] Webview loader does not load the hCaptcha

### ISSUE THREAD

**Issue by phauszzie** [inorichi/tachiyomi#3511]
#3406 # Device information
* Tachiyomi version: 0.9.2
* Android version: 9.0
* Device: Note 9

## Steps to reproduce
1. open any manga site that has hCaptcha using webview
2. page gets stuck loading the captcha and never finishes

### Expected behavior
Captcha should load and be able to be solved

### Actual behavior
Captcha never finishes loading

### Other details
The hCaptcha do not load in android webview or when webview implementer is set to chrome in android developer options. They work fine in the normal browser (chrome, firefox, adblock browser, samsung browser) though. Both webview implementer and chrome apps are the latest from the play store.

**Comment by arkon** [comment]
> open any manga site that has hCaptcha using webview

Like what?

I think this is actually fixed in the preview builds.

**Comment by phauszzie** [comment]
A few example sites: mangapark, mangaowl, mangahasu

**Comment by arkon** [comment]
Open again if still relevant in 0.10.x.

**Comment by Lightsockie** [comment]
~~Experiencing this currently with Mangahasu. I make it past the initial "checking your browser" step, but hang on the part the hCaptcha is supposed to load. Confirmed it as loading in the normal chrome browser.~~
```
App version: 0.10.5
Android version: 10
Device name: OnePlus8
```

Actually, I got it working. I was running webview version `85.0.4183.101`, but I've updated to `85.0.4183.127` and now it seems to be good. Gonna leave this comment here though still so other people that stumble in can know to make sure they update :^)
